src/store/page/saga.ts

```diff
@@ -29,6 +31,7 @@ const getPageByName = (name) => {
   const query = `
     query Page {
       ${pages[name]}
+      ${cartQuery}
```

Comment:
If required cart query in all pages for the website?

Reply:
This configuration was made for have cart data updated in page load